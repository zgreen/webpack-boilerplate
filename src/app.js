import React from "react";
import Hello from "./Hello";

class App extends React.Component {
  render() {
    return (
      <div>
        <Hello />
      </div>
    );
  }
}

export default App;
