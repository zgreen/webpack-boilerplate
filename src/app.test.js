/* globals expect test */
import { greet, greeting, helloWorld } from './app'

test('It returns "Hello, World."', () => {
  expect(helloWorld()).toBe('Hello, World.')
})

test('A greeting is rendering in the DOM.', () => {
  const app = document.createElement('div')
  app.id = 'app'
  document.body.appendChild(app)
  greeting()
  expect(app.innerHTML).toBe(`<h1>${helloWorld()}</h1>`)
})

test('Greet renders properly.', () => {
  expect(greet()).toMatchSnapshot()
})
