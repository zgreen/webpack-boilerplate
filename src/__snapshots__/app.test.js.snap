// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Greet renders properly. 1`] = `"<h1>Hello, World.</h1>"`;
